docs/components/badge.md

```diff
@@ -77,32 +80,40 @@ Using the `custom` appearance in conjunction with `color` and `background-color`
 | `--KBadgeDangerBorder`      |                               |
 | `--KBadgeDangerBackground`  |                               |
 
-An example of making the badges more like pills
+An example of making the badges more rad
```

Comment:
I mean, I like having fun text, but this doc is available for everyone, even outside of Kong. Do we want this text?

Reply:
Maybe something like: An example of theming the 'danger' appearance value.

Reply:
Are you trying to steal my joy @kaiarrowood